lib/ProductOpener/GeoIP.pm

```diff
@@ -68,6 +106,23 @@ sub get_country_for_ip {
 	return $country;
 }
 
+=head1 FUNCTIONS
+
+=head2 get_country_code_for_ip()
+
+C<get_country_code_for_ip()> takes the IP address as input paramter and returns the country iso code where IP address is located.
```

Comment:
```suggestion
C<get_country_code_for_ip()> takes the IP address as input parameter and returns the country iso code where IP address is located.
```

Reply:
Thank you so much for pointing this out!!
I have incorporated changes, Please check!!